### PR TITLE
Increase test coverage

### PR DESCRIPTION
**Pull request by jeremykubica**
Increase the test coverage through a combination of adding tests and marking some edge conditions as `# pragma: no cover`

In the process, I found the function `add_column_mapping` did not work as expected (and was untested). It has been removed.

### CODE REVIEW

File: tests/lightcurvelynx/test_graph_state.py [hunk 1]
```diff
@@ -570,6 +570,22 @@ def test_graph_state_equal():
     state6 = GraphState(num_samples=2)
     assert state5 != state6
 
+    # Two states are not equal if they have sets of parameters.
```
Comment (olivialynn) [inline tests/lightcurvelynx/test_graph_state.py]:
```suggestion
    # Two states are not equal if they have different sets of parameters.
```

**Review by olivialynn — APPROVED** [review]
